lib/rubocop/rspec/hook.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module RSpec
+    # Wrapper for RSpec hook
+    class Hook < Construct
+      def name
+        node.method_name
+      end
+
+      def unknown_scope?
+        return false unless scope_argument
+
+        !scope_argument.sym_type?
+      end
+
+      def scope
+        case scope_argument.to_a.first
+        when nil, :each, :example then :each
+        when :context, :all       then :context
+        when :suite               then :suite
+        else
+          raise "Unrecognized scope #{scope_argument.to_a.first}"
```

Comment:
Worth defining a rubocop-rspec error for this rather than a plain `RuntimeError`?